barman/postgres.py

```diff
@@ -1813,3 +1838,56 @@ def stop_exclusive_backup(self):
         return self._stop_backup(
             super(StandbyPostgreSQLConnection, self).stop_exclusive_backup
         )
+
+
+class PostgresKeepAlive:
+    """
+    Context manager to maintain a Postgres connection alive.
+
+    A child thread is spawned to execute heartbeat queries in the background
+    at a specified interval during its living context.
+
+    It does not open or close any connections on its own. Instead, it waits for the
+    specified connection to be opened in the main thread, making use of the same
+    current transaction.
```

Comment:
```suggestion
    current session.
```

Session is a more appropriate word in this context.